src/scaffoldmaker/annotation/stellate_terms.py

```diff
@@ -0,0 +1,25 @@
+"""
+Common resource for stellate annotation terms.
+"""
+
+# convention: preferred name, preferred id, followed by any other ids and alternative names
+stellate_terms = [
+    ( "stellate arm 1", None),
+    ( "stellate arm 2", None),
+    ( "stellate arm 3", None),
+    ( "stellate face 2-3", None),
+    ( "stellate face 1-2", None),
+    ( "stellate face 3-1", None),
```

Comment:
It's OK to put these here I guess, but I'm doubtful about whether they'll get proper ontology term IDs.
Also, if you support N-arms on the stellate you'll have a harder time managing these; you could have just created the names in the code according to this pattern.